### DISCUSSION TITLE
Feature Request: Alternative to isLoading for useSWRInfinite

### DISCUSSION THREAD

**Discussion by VladimirMikulic** [vercel/swr#2410] [Ideas]
## Description / Observed Behavior

`isLoading` only applies to the initial / first request.
I have a project with many infinite scrolls and this is the line which is repeated (taken from your infinite loading example):
```
const isLoadingMore =
    isLoadingInitialData ||
    (size > 0 && data && typeof data[size - 1] === "undefined");
```

`isLoadingMore` in this case is true while the next page data is being loaded and false otherwise:
- Request to /page/<n>
- isLoadingMore = true
- Request to /page/<n> done
- isLoadingMore = false

I was wondering if it would be possible to expose a property like `isLoadingMore` for `useSWRInfinite` so we avoid repeating this block for every infinite scroll in the app. Or maybe make `isLoading` to be true for all loading useSWRInfinite requests? (i.e. regardless of whether it's initial request or subsequent)

## Repro Steps / Code Example

https://swr.vercel.app/examples/infinite-loading

## Additional Context

SWR: v2.0.3

**Comment by devjiwonchoi** [comment]
Initiated this feature with #2728
